.pylintrc

```diff
@@ -120,10 +120,14 @@ disable=locally-disabled,
   c-extension-no-member,
   cyclic-import,
   isinstance-second-argument-not-valid-type,
-  bad-continuation
+  bad-continuation,
+  inherit-non-class
```

Comment:
Is there a way to disable this only for the relevant code? IIRC pylint checkers can be disabled per file.

Reply:
Yes, that would also be possible - but that would make it harder to adopt NamedTuple in other parts of Beancount. I think its better to have this globally disabled until this rule is adjusted for Python 3.9.

Reply:
I think some of these are obsolete anyway, bad-continuation isn't supported yet (they advise using Black instead).